bors.toml

```diff
@@ -1,3 +1,3 @@
-status = ["instant-meilisearch-tests", "instant-meilisearch-style-tests"]
+status = ["instant-meilisearch-tests", "instant-meilisearch-style-tests", "instant-meilisearch-playground-builds"]
```

Comment:
```suggestion
status = ['integration-tests', 'style-check', 'playground-builds']
```

Reply:
Can you rename the job like this please? It's more readable, too long name are not really convenient to read on the PR 🙂
Sorry for this commit suggestion, it's not really related to your PR, I can do it myself if you want. It also makes you change the name in the CI file.

Reply:
I can do it no problem